lib/data/repository/user_repository.dart

```diff
@@ -34,4 +37,20 @@ class UserRepository {
       throw errorMessage;
     }
   }
+
+  Future<List<User>> getUsersFromApi() async {
+    try {
+      final response = await _cobeApi.getUsers();
+      final userList =
+          (response.data as List).map((e) => User.fromJson(e)).toList();
+      _userBox.addAll(userList);
+      return userList;
+    } catch (e) {
+      debugPrint('Error during API call or JSON parsing: $e');
```

Comment:
If a network request fails, it would be nice to show users from cache as well, not just when there is no internet connection